.github/workflows/test.yml

```diff
@@ -0,0 +1,61 @@
+name: test
+on:
+  push:
+
+jobs:
+  test_with_workspaces_support:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # All currently supported LTS versions + latest with workspaces support
```

Comment:
Consider getting rid of this comment given it's a tad misleading with the same comment being below. Could you add some quick explanation on what this `--workspaces` means to the PR?

Reply:
I'll remove both comments (they are not entirely the same).
But I'll add a `MAINTENANCE.md` doc to explain how these files should be updated when future node versions are released.

Once we only support 16,18,20 they will be different and simpler.

Reply:
Any command run with `--workspaces` will run the command against individual packages but in a predictable order.

order is the same as the order in which workspaces are declared :
1. packages
2. plugins
3. plugin-packs

This ensures that dependencies for plugins are build first and that all plugins are build before handling `postcss-preset-env`

`--if-present` is added because some things might not have a script.
But I think it would be better to remove those flags in the future.
With them it is possible to forget `scripts.build` in a `package.json` and it would go undetected in CI.